sci-libs/rdkit/rdkit-2024.03.4.ebuild

```diff
@@ -33,12 +33,11 @@ RDEPEND="dev-libs/boost
 DEPEND="${RDEPEND}"
 
 PATCHES=(
-	"${FILESDIR}/${PN}-2021.09.4-find-rapidjson.patch"
+	"${FILESDIR}/find-rapidjson.patch"
```

Comment:
Patches should contain the `PN` and usually also the first version where it was introduced.